src/engine/Drawing/Animation.ts

```diff
@@ -118,7 +118,10 @@ module ex {
             
             this.naturalWidth = images[0] ? images[0].naturalWidth : 0;
             this.naturalHeight = images[0] ? images[0].naturalHeight : 0;
+
+            this.freezeFrame = images.length - 1;
```

Comment:
@eonarheim We should probably have a broader discussion about being able to edit the Sprites on an Animation directly, and the potential pitfall that could become. I'll leave it alone for now.